test/e2e/topology_updater_test.go

```diff
@@ -517,7 +517,7 @@ excludeList:
 
 				return true
 
-			}, 1*time.Minute, 10*time.Second).Should(BeTrue())
```

Comment:
Instead of increasing timeout you can try to set `sleepInterval` to 5s, it should make `topology-updater` to refresh NRTs every 5s.

Reply:
```
	flagset.DurationVar(&resourcemonitorArgs.SleepInterval, "sleep-interval", time.Duration(60)*time.Second,
		"Time to sleep between CR updates. zero means no CR updates on interval basis. [Default: 60s]")
```
It seems it defaults to 60s, hence the flakiness 😄

Reply:
Thanks @PiotrProkop, will do. I was sure somebody had a better idea than what I had 😅

Reply:
Done